src/test/java/com/dropwizard/example/h2/model/PersonTest.java

```diff
@@ -0,0 +1,62 @@
+package com.dropwizard.example.h2.model;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertTrue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * The Person test class.
+ */
+public class PersonTest {
+
+    private Person person;
+
+    @Before
+    public void setUp(){
+        person = new Person();
+    }
+
+    @Test
+    public void test_constructor(){
```

Comment:
Tests should be named like this _testConstructor_. No underscores

- [ ] Rename test methods.